openmrs-19/src/main/java/org/motechproject/openmrs19/util/ConverterUtils.java

```diff
@@ -66,7 +66,22 @@ public static OpenMRSPerson toOpenMRSPerson(Person person) {
         converted.setGender(person.getGender());
 
         if (person.getPreferredAddress() != null) {
-            converted.setAddress(person.getPreferredAddress().getAddress1());
+            converted.setAddress(
```

Comment:
1. use StringBuilder
2. move converting method to Person model
3. add unit test